src/intTest/resources/edifact/multi_pathology.edifact.dat

```diff
@@ -0,0 +1,40 @@
+UNB+UNOC:3+XX11+000000024600002:80+100301:1751+1015++MEDRPT++1'
```

Comment:
the interchange start example we have is slightly different.

`UNB+UNOC:3+000000004400001:80+000000024600002:80+100301:1751+1015++MEDRPT++1'`

This includes `:80` after the recipient. I've removed this for now, until we've updated the interchange header parser.